src/domain/httpsactuatorserver.py

```diff
@@ -0,0 +1,95 @@
+"""HTTPS Actuator server module"""
+
+import json
+import logging
+import re
+import ssl
+import threading
+from http import HTTPStatus
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+from actuator import Actuator
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPActuatorRequestHandler(BaseHTTPRequestHandler):
+    def do_GET(self):
+        if re.search("/api/v1/actuators/*", self.path):
```

Comment:
Since this is called for every GET request, it's better to compile the regex and put it at the GLOBAL module level.